blog/2025-07-16-updated-pattern-for-helpers-linking-to-devices.md

```diff
@@ -0,0 +1,29 @@
+---
+author: Erik Montnemery
+authorURL: https://github.com/emontnemery
+title: "Updated guidelines for helper integrations linking to the source entity's device"
+---
+
+Helper integrations should not add their own config entries to the source entity's device or to a user selected device. Instead, they should just link their entities to the device of the source entity.
+
+Adding the helper config entry to another integration's device is no longer supported and will stop working in Home Assistant Core 2026.8.
```

Comment:
Why is this deprecation period so short?

Reply:
One year is not that short?